admin/modules.php

```diff
@@ -215,6 +215,10 @@
                           }
                         }
                         $module_info['keys'] = $keys_extra;
+                        if (method_exists($module, 'verify')) { 
+                          $module_info['configuration_errors'] = $module->verify(); 
```

Comment:
Both instances of `verify` need to be changed to `get_configuration_errors`

Reply:
got 'em.